geometry_msgs/Pose goal_pose
---
int8 success